Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGL.cpp

```diff
@@ -127,11 +127,11 @@ void RemoteGraphicsContextGL::workQueueInitialize(WebCore::GraphicsContextGLAttr
 {
     assertIsCurrent(workQueue());
     platformWorkQueueInitialize(WTFMove(attributes));
-    m_streamConnection->open(workQueue());
+    m_streamConnection->open(protectedWorkQueue());
     if (m_context) {
         m_context->setClient(this);
-        String extensions = m_context->getString(GraphicsContextGL::EXTENSIONS);
-        String requestableExtensions = m_context->getString(GraphicsContextGL::REQUESTABLE_EXTENSIONS_ANGLE);
+        String extensions = protectedContext()->getString(GraphicsContextGL::EXTENSIONS);
```

Comment:
Instead of calling protectedContext() twice, we should create a Ref/RefPtr local variable.

Reply:
Fixed!